.gitignore

```diff
@@ -0,0 +1,3 @@
+*.DS_Store
```

Comment:
please create a root level folder called dnstwist and move all files underneath. required by the automation script that generates rpms